core/math/bsp_tree.cpp

```diff
@@ -577,12 +577,11 @@ BSP_Tree::BSP_Tree(const PoolVector<Face3> &p_faces, real_t p_error_radius) {
 	error_radius = p_error_radius;
 }
 
-BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius) {
-
-	nodes = p_nodes;
-	planes = p_planes;
-	aabb = p_aabb;
-	error_radius = p_error_radius;
+BSP_Tree::BSP_Tree(const Vector<Node> &p_nodes, const Vector<Plane> &p_planes, const Rect3 &p_aabb, real_t p_error_radius)
```

Comment:
Actually there are 3 settings related to `ConstructorInitializer` which you could play with and see what comes out of it: https://github.com/godotengine/godot/blob/master/.clang-format
```
# BreakConstructorInitializersBeforeComma: false
# ConstructorInitializerAllOnOneLineOrOnePerLine: false
# ConstructorInitializerIndentWidth: 4
```

See http://releases.llvm.org/3.9.1/tools/clang/docs/ClangFormatStyleOptions.html for the applicable documentation.

Actually in clang-format 6 we'll have a much cooler option: http://clang.llvm.org/docs/ClangFormatStyleOptions.html namely `BreakConstructorInitializers`, but for now we're recommending clang-format 3.9.x which is more widespread on Linux distros.